Controllers/HomeController.cs

```diff
@@ -17,9 +22,12 @@ public HomeController(ILogger<HomeController> logger, WidgetService widgetServic
             _widgetService = widgetService;
         }
 
+
         public IActionResult Index()
         {
-            return View();
+            object getAllWidgets = _widgetService.GetAllWidgets();
```

Comment:
The variable "getAllWidgets" here is assigned the list of widgets returned by the function.  The next line sets the "list" variable equal to the same.  You should eliminate the "getAllWidgets" variable and assign the list returned by the function directly to the "list" variable on line 29.

Reply:
Removed the redundant variable and renamed getallwidgets to list.

Reply:
This looks great now!